packages/sources/metalsapi/test/integration/__snapshots__/adapter.test.ts.snap

```diff
@@ -281,7 +73,7 @@ exports[`execute latest api should error with batched base symbols 1`] = `
 Object {
   "error": Object {
     "feedID": "[BTC|XAU]/USD",
-    "message": "Base symbol XAU,BTC is not batchable.",
```

Comment:
@austinborn I am surprised that this error message did not end up as the type validation one.

Reply:
Until the input params are updated for an adapter, the only validation they go through is checking that the param exists if `inputParameters.[param]` is `true` or an alias array. Did you add the new `inputParameters` or did that come from some other commit?

Reply:
I added them here with the type property